docs/src/net_activation.md

```diff
@@ -0,0 +1,66 @@
+# [Supported Layers and Activation Functions](@id layers_activation)
+
+The PEtab SciML neural network model YAML format supports numerous standard neural network layers and activation functions. Layer names and associated keyword arguments follow the PyTorch naming scheme. PyTorch is used because it is currently the most popular machine learning framework, and its comprehensive documentation makes it easy to look up details for any specific layer or activation function.
+
+If support is lacking for a layer or activation function you would like to see, please file an issue on [GitHub](https://github.com/sebapersson/petab_sciml/issues).
+
+## Supported Neural Network Layers
+
+The table below lists the supported and tested neural network layers along with links to their respective PyTorch documentation. Additionally, the table indicates which tools support each layer.
```

Comment:
No need to repeat IMO. Move to the previous section and delete the duplicate in the next section. The section titles make the difference between the tables clear.